frontend/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14.17.3-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install --silent
+RUN npm install react-scripts@4.0.3 -g --silent
+
+EXPOSE 3000
+# add app
+COPY . ./
+
+# start app
+CMD ["npm", "start"]
```

Comment:
It would be much, much better if you run the **built** app (e.g. with Nginx). This can be done in a separate PR, just keep it in mind.